client/my-sites/email/email-management/email-home.tsx

```diff
@@ -88,9 +89,9 @@ const EmailHome = ( props: EmailManagementHomeProps ) => {
 		selectedIntervalLength,
 		sectionHeaderLabel,
 		source,
-	} = props;
+	}: EmailManagementHomeProps = props;
```

Comment:
Just dropping by quickly to say that this type definition is actually redundant. Typescript is already aware that `props` has the `EmailManagementHomeProps` type and the destructured variables will preserve the appropriate types.